src/jit/hwintrinsicxarch.cpp

```diff
@@ -324,7 +365,10 @@ GenTree* Compiler::impSSEIntrinsic(NamedIntrinsic intrinsic, CORINFO_METHOD_HAND
     switch (intrinsic)
     {
         case NI_SSE_IsSupported:
-            retNode = gtNewIconNode(featureSIMD && compSupports(InstructionSet_SSE));
+            // TODO - remove "JitConfig.EnableIncompleteISAClass()" after fully implement hardware intrinsics of this
+            // class
+            retNode =
+                gtNewIconNode(JitConfig.EnableIncompleteISAClass() && featureSIMD && compSupports(InstructionSet_SSE));
```

Comment:
Isn't this test redundant with the one above? The check for `JitConfig.EnableIncompleteISAClass()` can only be done under `DEBUG`, but I believe that this check is redundant anyway, unless there's something I've missed.
Also, I still think these "TODO" comments are unnecessary, as I believe that we will need that facility for some time as new ISA feature sets are added and implemented over time.

Reply:
> Isn't this test redundant with the one above?

No, this check lets `IsSupported` return `false` for incomplete classes, but the above one lets intrinsics throw PNSE when the call site is not guarded by `IsSupported`.

> The check for JitConfig.EnableIncompleteISAClass() can only be done under DEBUG,

Oops, my mistake. Will fix.